src/utils/constants/routes.ts

```diff
@@ -6,6 +6,6 @@ export const INFO = '/info';
 export const BLOCKS = '/blocks';
 export const NOT_FOUND = '/not-found';
 
-export const TRANSACTION_DETAILS = '/tx';
+export const TRANSACTION_DETAILS = '/transaction';
```

Comment:
Can you change it to /transcations as well?

Reply:
Reverted to `/tx`